.github/ISSUE_TEMPLATE/contributor-nomination.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Contributor Nomination
+about: Nominate a GitHub user for additional rights on the Zephyr Project
+title: ''
+labels: Role Nomination
+assignees: ''
+
+---
+
+# Background
+
+The [TSC Project Roles] defines the main roles for the Zephyr Project, including
+Maintainter, Collaborator, and Contributor.
+
+TODO: define new roles for Contributors with extra rights? This would correspond
+to the Github "Triage" permission level. This template uses Contributor+ as a
+placeholder.
```

Comment:
Process WG:

- keep Contributor level; do not introduce Contributor+ by that name or any other
- add a line to the documentation for Contributor which describes the process for opening an issue that fills out this template